watch_node.py

```diff
@@ -0,0 +1,58 @@
+# @todo - this should be combined with build_from.py
+import argparse
+import codecs
+
+try:
+    import requests_cache
+    requests_cache.install_cache('fr_cache')
+except ImportError:
+    # If the cache library isn't present, do nothing -- we'll just make full
+    # HTTP requests rather than looking it up from the cache
+    pass
+
+from regparser.builder import Builder
+from regparser.notice.changes import node_to_dict, pretty_change
+from regparser.tree.struct import find
+
+
+def init_reg_and_builder(args):
+    """This function repeats a lot of what's done in build_from.py -- parse
+    the provided regulation file into a tree and create a Builder object"""
+    reg_text = ''
+    with codecs.open(args.filename, 'r', 'utf-8') as f:
```

Comment:
Should this share code with build_from? So that, if we ever change how that works, this will change too?